docs/reference/01-scnode-api-spec.rst

```diff
@@ -1415,61 +1685,61 @@ ____________
 
    .. tab:: Bash
 
-      curl -X POST "http://127.0.0.1:9086/mainchain/blockReferenceByHash" -H "accept: application/json" -H "Content-Type: application/json" -d "{\"hash\":\"string\",\"format\":false}"
+      curl -X POST \"http://127.0.0.1:9086/mainchain/blockReferenceByHash\" -H \"accept: application/json\" -H \"Content-Type: application/json\" -d \"{\\\"hash\\\":\\\"string\\\",\\\"format\\\":false}\"
```

Comment:
Just a consideration, absolutely not a criticism or a blocker (because your way to do it it's perfect) : I think that, in order to simplify a little, it should be possible to do : 

`curl -X POST "http://127.0.0.1:9086/mainchain/blockReferenceByHash" -H "accept: application/json" -H "Content-Type: application/json" -d '{"hash":"string","format":false}'`

Usually this way works in my experience.